util/rich-indexer/src/indexer/remove.rs

```diff
@@ -229,11 +229,11 @@ async fn script_exists_in_output(
         "#,
     )
     .bind(script_id)
-    .fetch_one(&mut *tx)
+    .fetch_one(tx.as_mut())
     .await
     .map_err(|err| Error::DB(err.to_string()))?;
 
-    Ok(row_lock.get::<bool, _>(0) || row_type.get::<bool, _>(0))
+    Ok(row_lock.get::<i64, _>(0) == 1 || row_type.get::<i64, _>(0) == 1)
```

Comment:
Perhaps we can return early with =Ok(row_lock.get::<i64, _>(0) == 1)= after immediately fetching =row_lock=?